recordium/utils.py

```diff
@@ -26,11 +25,17 @@ def _ensure_dir_exists(dirpath):
 
 
 # simple attributes to have this calculated at one place only
-config_basedir = BaseDirectory.xdg_config_home
-cache_basedir = os.path.join(BaseDirectory.xdg_cache_home, 'recordium')
-_ensure_dir_exists(cache_basedir)
-data_basedir = os.path.join(BaseDirectory.xdg_data_home, 'recordium')
-_ensure_dir_exists(data_basedir)
+
+
+app = appdirs.AppDirs("recordium")
+for any_dir in [app.user_cache_dir,
+                app.user_config_dir,
+                app.user_data_dir]:
+    _ensure_dir_exists(any_dir)
+
+data_basedir = app.user_data_dir
+cache_basedir = app.user_cache_dir
+config_basedir = app.user_config_dir
```

Comment:
Note that we're changing the config directory here; so if a user has already configured system, you will be missing that original file.

So, if you want to change the directory here, you should move the original file in.